### ISSUE TITLE
improve(prodUrl): empty outDir leads to wrong assets path 

### ISSUE THREAD

**Issue by dennisbaum** [OblikStudio/kirby-vite#1]
If i build into the same (public)-directory or in my case the hole site into a dist-folder with the same structure, 
i have to config an empty outDir - otherwise no manifest.json will be found nor the assets.

`
'oblik.vite.build.outDir' => '',
`

If i do so, it leads to a wrong asset path: ( please note the two slashes

![grafik](https://user-images.githubusercontent.com/562826/168822688-c4d1ef15-b206-4403-ad4b-6ea7da5fc563.png)

i would like to filter the path-array to empty strings by strlen. 
so the empty `outDir` wont be included in the array, and the path is correct.

```php
public function prodUrl(string $path)
{
    return implode('/', array_filter([
        site()->url(),
        option('oblik.vite.build.
        $path
    ], 'strlen'));
}
```

what do you think?
Thx!

**Comment by yandodov** [comment]
Yep, you can do that. That's actually how I made it initially, but thought that having an `outDir` was conventional, so I thought that I don't need to support it. Feel free to create a PR!

**Comment by yandodov** [comment]
@dennisbaum I published [version 1.0.1 on Packagist](https://packagist.org/packages/oblik/kirby-vite#1.0.1) with the fix. Let me know if everything works fine.
